components/automate-ui/src/app/modules/infra-proxy/infra-search-bar/infra-search-bar.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="search-wrapper" tabindex="0">
+  <div class="search-input-wrapper">      
+    <input #search_box chefInput
+      id="search_box"
+      aria-label="Search"
+      (keyup)="handleInput($event.key, search_box.value)"
+      [value]="inputText"
+      type="text"
+      class="search-input"
+      placeholder="{{ getFilterText() }}"
```

Comment:
In a template it is important to avoid function calls _in rendered text_ unless absolutely necessary.
Angular cannot know a priori whether the result of a function varies from one digest cycle to the next so it will be calling the function many more times than necessary.
This particular case lends itself to a simple solution:
```suggestion
      placeholder="Search {{ placeHolder }}"
```